docs/release-notes/2.26.0.md

```diff
@@ -0,0 +1,91 @@
+# Percona Monitoring and Management 2.26.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">January 13, 2022</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+- **Experimental Dashboards**
+
+New experimental dashboards are introduced that will be shipped to the PMM users. These dashboards will be uploaded to the Experimental folder to enable the users to test them in their environment and provide feedback.
+ 
+ The following Dashboards are being shipped as part of this release:
+
+ - Dbaas: [K8s monitoring dashboard](https://www.percona.com/doc/percona-monitoring-and-management/2.x/details/dashboards/dashboard-cluster-summary.html)
```

Comment:
@rnovikovP we don't have this in 2.26.0, do we?

Reply:
yes, we do - disregard.

Reply:
This is about  https://jira.percona.com/browse/PMM-9003